wcivf/apps/elections/tests/test_helpers.py

```diff
@@ -31,3 +35,56 @@ def test_with_territory_code_malformed_id(self):
         expected = expected_sopn_publish_date("whoknows", "ENG")
 
         assert expected is None
+
+
+class TestEEHelper:
+    @pytest.fixture
+    def ee_helper(self):
+        return EEHelper()
+
+    def test_base_elections_url(self, ee_helper):
```

Comment:
It's possible for this to fail in some situations - for example, I have changed `settings.EE_BASE` in my `local.py` in the past when testing the relationship between local EE and local WCIVF. In that case, `            ee_helper.base_elections_url` would return `http://localhost:8001/api/elections/`. 

I expect it's worth adding a [settings override](https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.override_settings) here just in case.